File: 백준/Bronze/2884. 알람 시계/알람 시계.js
const fs = require('fs');
const input = fs.readFileSync(0, 'utf-8').trim().split(' ');

let h = parseInt(input[0]);
let m = parseInt(input[1]);

m -= 45;

if (m < 0) {
  h -= 1;
  m += 60;
}

if (h < 0) {
  h += 24;
}

console.log(`${h} ${m}`);

File: 백준/Bronze/2884. 알람 시계/알람 시계.swift
let input = readLine()!.split(separator:" ").map{Int($0)!}
var H: Int = input[0]
var M: Int = input[1]

M -= 45
if (M < 0) {
    M += 60
    H -= 1
    if (H < 0) { H += 24 }
}
print("\(H) \(M)")